polling_stations/apps/data_collection/management/commands/__init__.py

```diff
@@ -40,159 +38,209 @@
 from addressbase.helpers import create_address_records_for_council
 
 
-class CsvHelper:
+class Slugger:
```

Comment:
Can this have some tests? Just to ensure it's dealing with the "1/2" vs "12" issue properly, and future ones we might come across. Seems like a high chance of hard to initially spot regression in future versions otherwise.